### PR TITLE
Updates to Gruntfile to allow for a CD workflow

### PR DESCRIPTION
**Pull request by mmcc**
Originally the plan was for this to be just a PR to break out commands in the Gruntfile, but this has turned into a PR to allow us to use our current contrib release with a continuous deployment service (Codeship). The idea being that we can now simply push to specific branches to trigger a new release.

TL;DR
- Broke tasks into their own files
- Removed a few unused variables and require statements
- Added new tasks for creating a Github release
- Updated the contrib file to reflect said changes

### CODE REVIEW

**Comment by mmcc** [comment]
@videojs/contributors I think this is good to go. Thoughts? It'd be nice to pull this in soon so we can start making other changes to the Gruntfile (such as updating our S3 plugin) without having to duplicate them here every time.

**Comment by gkatsev** [comment]
Looking

**Comment by heff** [comment]
lgtm!

File: Gruntfile.js [hunk 1]
```diff
@@ -333,6 +370,8 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-version');
   grunt.loadNpmTasks('grunt-tagrelease');
   grunt.loadNpmTasks('chg');
+  grunt.loadNpmTasks('grunt-fastly');
+  grunt.loadNpmTasks('grunt-github-releaser');
```
Comment (gkatsev) [inline Gruntfile.js]: we should just get [load-grunt-tasks](https://www.npmjs.com/package/load-grunt-tasks)

**Comment by gkatsev** [comment]
Other than `load-grunt-tasks`, LGTM

File: Gruntfile.js [hunk 2]
```diff
@@ -312,6 +333,22 @@ module.exports = function(grunt) {
       commit:  true,
       message: 'Release %version%',
       prefix:  'v'
+    },
+    'github-release': {
```
Comment (dmlap) [inline Gruntfile.js]: I had no idea this task existed. Awesome!

File: contrib.json [hunk 3]
```diff
@@ -202,8 +202,11 @@
         [ "git pull upstream stable",                "Update the release branch" ],
         [ "npm install",                             "Ensure dependency updates have been installed" ],
         [ "grunt test",                              "Run tests" ],
-        [ "grunt chg-release:{{ release_type }}",    "Update the changelog with the new release" ],
         [ "grunt version:{{ release_type }}",        "Bump package versions" ],
+        { "exec": "grunt vjs-version",
+          "id": "version",                   "desc": "Return the current VJS Version from the package.json file"},
+        [ "grunt chg-release:{{ version }}",         "Update the changelog with the new release" ],
+        [ "grunt clean:dist",                        "Clean out the dist folder before the build" ],
```
Comment (dmlap) [inline contrib.json]: It would be nice to clean out build/ as well. We have a number of vjs releases in npm with old video.js zips in the build folder.

**Comment by dmlap** [comment]
My one comment is a "nice-to-have". LGTM.

**Comment by mmcc** [comment]
Closed via 0b392fa9f5

**Pull request closed without merging**